docs/java/features/odata/type-safe-client-odata-v4.mdx

```diff
@@ -198,10 +198,62 @@ service.getAllBusinessPartner()
     .execute(destination);
 ```
 
+<!--
+### Batch Requests 
+Batch requests allow wrapping multiple OData requests into one single batch call. Thereby we reduce the number of roundtrips to the remote server.
+ Refer to the [official OData V4 spec](http://docs.oasis-open.org/odata/odata/v4.0/os/part1-protocol/odata-v4.0-os-part1-protocol.html#_Toc372793748) 
+for further reference about batch requests, their semantics and the request + response format.
+
+:::warning
+The described API is subject to change
+:::
+
+#### Execute Batch Request
+The `service` object offers the method `batch` which allows access to the batch-related methods that help build the batch request.
+
+Multiple single requests (like `createBusinessPartnerAddress`) can be wrapped into so-called change sets.
+Each change set is opened with `beginChangeSet` and closed with `endChangeSet`. You can wrap multiple change sets into one batch request.
+
+Use `executeRequest` to issue the batch request to the remote system. We receive an instance of `BatchResponse` as result object.
+
+```java
+final BatchResponse result =
+    service
+        .batch()
+        .beginChangeSet()
```

Comment:
We're aware that API for V4 will look different, but this is being left as placeholder for the actual documentation.